.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
+    "align",
+    "background",
+    "bgcolor",
+    "border",
+    "frameborder",
+    "style"
+  ],
```

Comment:
There's no need to add this file. Also, don't add package-lock.json and package.json.

Reply:
okay, got it! thx for the advice and thx for the review!